scripts/xenadmin-build.ps1

```diff
@@ -317,15 +317,19 @@ $xmlFormat=@"
 
 $msi_url = $XC_UPDATES_URL -replace "XCUpdates.xml","$appName.msi"
 $date=(Get-Date).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")
-$productVersion =  "$BRANDING_XC_PRODUCT_VERSION.$buildNumber"
-$productFullName =  "$appName $productVersion"
+$productVersion = "$BRANDING_XC_PRODUCT_VERSION.$buildNumber"
+$productFullName = "$appName $productVersion"
 
-Write-Host "INFO: Generating XCUpdates.xml"
+$prodXml = [string]::Format($xmlFormat, $productFullName, $date, $msi_url, $msi_checksum, $productVersion)
+$testXml = [string]::Format($xmlFormat, $productFullName, $date, "@DEV_MSI_URL_PLACEHOLDER@", $msi_checksum, $productVersion)
 
-[string]::Format($xmlFormat, $productFullName, $date, $msi_url, $msi_checksum, $productVersion) |`
-    Out-File -FilePath $OUTPUT_DIR\XCUpdates.xml -Encoding utf8
+Write-Host $prodXml
+Write-Host $testXml
 
-Write-Host "INFO: Generating stage-test-XCUpdates.xml. URL is a placeholder value"
+$bomLessEncoding = New-Object System.Text.UTF8Encoding $false
```

Comment:
can we add some context here in the form of a comment? someone simply reading this code won't understand why we're doing this